Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM debian:10-slim
```

Comment:
Any reason for debian? Alpine is generally more lightweight.

Reply:
a mix of bad habits that made it too time-consuming to rebase it. Basically no local dockerbuilds, using github to edit and redownloading from quay on every change. At the time of finishing the sidecar work I didn't wanna see it again in my life. 

TODO: rebase and use alpine instead of debian.